justfile

```diff
@@ -15,7 +15,17 @@ _rustflags := env_var_or_default("RUSTFLAGS", "")
 
 # If we're running in Github Actions and cargo-action-fmt is installed, then add
 # a command suffix that formats errors.
-_fmt := if env_var_or_default("GITHUB_ACTIONS", "") != "true" { "" } else {
+#
+# Clippy version also gets -Dwarnings.
+_fmt_clippy := if env_var_or_default("GITHUB_ACTIONS", "") != "true" { "-- -Dwarnings" } else {
+    ```
+    if command -v cargo-action-fmt >/dev/null 2>&1; then
+        echo "--message-format=json -- -Dwarnings | cargo-action-fmt"
+    fi
+    ```
+}
```

Comment:
yeah, this is a bit unfortunate, but i can't really think of a better solution...hmm.

Reply:
we could alternatively have separate `{{ _msg_fmt_json }}` and `{ _fmt }}` variables but that seems worse...